pages/faq.rst

```diff
@@ -164,6 +164,23 @@ I don’t want to use Elasticsearch as my backend storage system – can I use a
 
 You can, but we don’t suggest you do. You will not be able to use our query functionality or our analytic engine on the dataset outside the system. We only recommend another database if you want it for secondary storage.
 
+Can i add a restricted user to Graylog, to inlcude internal Graylog metrics in my Monitoring?
+---------------------------------------------------------------------------------------------
+
+You can create a user that has access only to ``/system/metrics`` at the Graylog API, this way it will be possible for you to integrate Graylog internal metrics to your monitoring.
```

Comment:
It's also possible without an additional user. Maybe we should include the reasoning behind it (using only restricted users to minimize impact if the credentials leaked).